src/pages/OnBoarding/util/OnBoardingStep.styled.js

```diff
@@ -188,3 +188,13 @@ export const NextButton = styled(SaveButton)`
       }
     `}
 `
+
+export const Spinner = styled.div`
```

Comment:
Nice spinner but unfortunately we don't use spinners.

We try and use skeleton shimmers instead. In this case it would be the addon placeholder cards with the shimmer effect over them. We actually already have the number of addon placeholders to show from `releases` data, which is nice.

`${getShimmerStyles()}`

<img width="636" alt="image" src="https://github.com/ynput/ayon-frontend/assets/49156310/dc3f7e53-9f68-4036-93ec-f479f14f4e07">

Reply:
Added shimmer effect. This is now how it looks now when loading state is true.


https://github.com/ynput/ayon-frontend/assets/16327908/02d6f0e6-509b-4aec-9aea-a57ad2e8d221